e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -0,0 +1,342 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "Region": {
+            "type": "string",
+            "metadata": {
+                "description": "The main region for resources created by this template."
+            }
+        },
+        "FarRegion": {
```

Comment:
What do you think about defaulting this to southeastasia?

Reply:
It is already hardcoded to 'southeastasia' in the .ps1. We can add an option to option to override if needd.